lib/ukrand/include/sys/random.h

```diff
@@ -32,6 +32,7 @@
 
```

Comment:
Ok so yeah I don't really like this file at all. It is a copy-paste to begin with. I would like to have [musl's random.h](https://github.com/bminor/musl/blob/master/include/sys/random.h) (which is also probably the origin of this copy-paste) as a file of ours in `lib/nolibc/musl-imported/include/sys/random.h` (just like the others there) instead.

This should be, however, a more general cleanup separate commit. If you'd like you could push an additional commit into this PR deleting this file and adding the one I suggested. Otherwise we could leave it for later, no problem, won't stop me from approving this PR.